packs/kubernetes/requirements.txt

```diff
@@ -0,0 +1 @@
+pykube
```

Comment:
probably choose a specific version unless you plan to maintain back compat with the pack